### DISCUSSION TITLE
How can I use other models?

### DISCUSSION THREAD

**Discussion by ItsMEMEGuy** [azuritecoin/OnnxDiffusersUI#8] [Q&A] [answered]
Hey,
I am very new to this stuff, so please go light on me.

I'm quite unsure of how to use other models with the UI.
Where can I find other models on the huggingface website? I found this site but I'm not sure if it is the right one: https://huggingface.co/models?other=stable-diffusion

Where do I look to get the .ckpt file? In the guide, it says that you used waifu diffusion. So on the website, that I gave just now, I went to the first result that said waifu diffusion (https://huggingface.co/hakurei/waifu-diffusion), and I can't find a .ckpt anywhere.
 
But then again you said, "IF the model is a .ckpt file". What if not? How do I recognize that?

Again sorry for the many questions. I am new to this and want to learn more about it so help would be much appreciated.

Thanks

**Comment by azuritecoin** [comment, marked as answer]
In short, if the model is hosted on the huggingface website **and** it says "diffusers" in their model card page then you only need to run the first conversion script. Just change the `--model_path` to point to the correct model name.

Another way to check is to go to the huggingface website for the model and browse the files. If the files have a bunch of folders (eg. shedulder, text_encoder, etc.) then again you can just use the first conversion script.

If there's only a `.ckpt` file present, then you'll need to do both conversion script as described in the guide.

Here's another site with more models, most of which are `.ckpt`: <https://rentry.org/sdmodels>

**Reply by ItsMEMEGuy** [reply]
Thanks for the quick reply.

Thank you for the link. It's really helpful and seems like it could have everything I would need. But just so I don't have any issues in the 
future: If I understand you correctly I should be able to download the files in one folder, put it in my folder for stable diffusion, and use:

`python convert_stable_diffusion_checkpoint_to_onnx.py --model_path="./waifu_diffusion" --output_path="./waifu_diffusion_onnx"`

right?

Luckily with the link, you gave I can just follow the guide so this question isn't as important, just curious if I ever found an interesting model with this type of file assortment.

Thanks again for the answer it really helped.

**Reply by azuritecoin** [reply]
Yes, the `convert_stable_diffusion_checkpoint_to_onnx.py` will do exactly that. And you have the right command.

**Reply by ItsMEMEGuy** [reply]
Okay, thank you very much for how quick you answered my questions.

**Reply by amida168** [reply]
I was able to use the midjourney-v4-diffusion model. 

https://www.kombitz.com/2022/11/09/how-to-use-midjourney-v4-diffusion-model-on-onnxdiffusersui/
